docs/overview/why-saleor/extensibility.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Extensibility model
+---
+
+### API Only
+
+• **Webhooks.** You can subscribe to more than a hundred [events](/developer/extending/webhooks/overview) to receive data from Saleor.
+
+• **API extensions.** Saleor can [proxy](developer/extending/webhooks/synchronous-events/overview) calls to your service to modify behavior or resources.
+
+• **Dashboard extensions.** [Extend admin](/developer/extending/apps/overview) functionality using any front-end stack embeddable into _iframes_.
+
+• **Granular permission control.** [Granular permissions](/developer/permissions) for access tokens.
+
+• **GraphQL oriented.** Provide GraphQL queries to shape the data provided via Webhooks.
+
+### Why not plugin architecture?
+
+Plugin architecture is popular among open-source projects, classical examples would be Magento or Shopify.
+Such a model has direct coupling which the platform which can be convenient to get started quickly,
```

Comment:
Rewrite proposal:

Direct coupling, as seen in that model, may offer a quick start, but it can become limiting and inflexible when compared to the loosely-coupled approach provided by extensibility via API/Webhooks.

Reply:
I don't think "limiting or inflexible" are the exact problem with direct coupling